akka-remote/src/main/protobuf/ContainerFormats.proto

```diff
@@ -26,4 +26,18 @@ enum PatternType {
 message Selection {
   required PatternType type = 1;
   optional string matcher = 2;
-}
\ No newline at end of file
+}
+
+message Identify {
+  required Payload messageId = 1;
+}
+
+message ActorIdentity {
+  required Payload correlationId = 1;
+}
+
+message Payload {
```

Comment:
@patriknw - I have only applied `Payload` to the new message types so the it's backward compatible on the wire. So I'm not using this for `SelectionEnvelope` although technically I could.